src/controllers/verify.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { config } from "dotenv";
+import { Request, Response } from "express";
+import jwt from "jsonwebtoken";
+import User from "../models/user";
+
+config();
+
+export const verifyAccount = async (req: Request, res: Response) => {
```

Comment:
This operation should be under user controller not a separate controller. Move it under user controllers or make it as a middleware.

Reply:
because everyone was working under the same file I was minimizing conflicts. but I can take it back in user controllers.